devfile.yaml

```diff
@@ -1,57 +1,21 @@
 apiVersion: 1.0.0
 metadata:
   name: che-server-in-che
-projects:
-  - name: che
-    source:
-      type: git
-      location: 'https://github.com/eclipse/che.git'
 components:
   - type: dockerimage
-    image: 'quay.io/eclipse/che-dev:nightly'
-    alias: che-server-dev
+    image: quay.io/eclipse/che-java11-maven:nightly
+    alias: maven
     mountSources: true
-    memoryLimit: 5Gi
+    memoryLimit: 2Gi
     volumes:
       - name: maven
         containerPath: /home/user/.m2
-  - type: dockerimage
-    image: 'quay.io/eclipse/che-server:nightly'
-    alias: che-server-runtime
```

Comment:
Originally it was possible to build and start Che server in dev workspace. You can see the instructions in [Che server contribution section](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#che-server-aka-ws-master).
Are we going to drop the ability to run Che server in dev workspace?

Reply:
Current devfile does not work at all. We'd like to rebuild it from scratch, because we're in different position that at the time current devfile was made. This PR should be the baseline from which we will iterate. It's just a first step which defines simple build without tests, because that something that we made working in time we've invested.

Reply:
I believe this might be a little bit outdated at this time. We want to iterate with small steps and fix `running` section in the next iteration.